framework/src/helper/function-analyzer.ts

```diff
@@ -96,6 +96,7 @@ const parseParameterString = (parameter: string,): Param => {
     case 'string':
     default:
       if (value.default === '') {
+        value.value = getEnv(value.envName, value.default);
```

Comment:
Please remove the duplication of code, this is likely going to lead to future bugs